README.md

```diff
@@ -30,54 +30,39 @@
 
 ## Introduction
 
-Growing numbers of people are working in startups and fast-growing tech companies.
-If you’re one of them, or are considering becoming one, you’ve probably experienced or
-tried to learn about stock and stock options, RSUs, and other types of
-[**equity compensation**](http://www.investopedia.com/terms/e/equity-compensation.asp).
+Businesses use a variety of methods to attract and retain the best people in their field. Some employers offer big salaries and bonuses, hip offices, work-from-home time, generous vacation, and even fancy on-site meals. Of all these methods, however, [**equity compensation**](http://www.investopedia.com/terms/e/equity-compensation.asp) is unique in its complexity, ability to align company and employee interests over a long period of time, and potential for wealth creation—the first ~3,000 employees of Facebook split roughly $23 billion, [according to The Financial Times](https://www.ft.com/content/d6599ae0-5738-11e1-869b-00144feabdc0).
 
-🚧 Groundwork:
-Significance (#s of ppl dealing with these questions, examples of real-world results
-(happy and horror) of taking equity comp, eg., why should we learn more, why should we
-care?)
+We *can not* guarantee—by any means—that reading this Guide will result in fortune smiling upon you as warmly as it did the early hires of Facebook. On the other hand, we firmly believe you will be able to walk away from reading this Guide with knowledge that will enable you to avoid making very costly mistakes.
```

Comment:
Can tighten a little:

"You will, however, be able to walk away knowing enough to avoid some of the most common, costly mistakes."

Elide "by any means".